doc/integrations/parlAI/application.py

```diff
@@ -41,7 +41,7 @@
 
 
 ACCESS_KEY = "AKIAtEpiGWUcQIelPRlD1Pi6xQ"
```

Comment:
ACCESS_KEY also need to be parameterized

Reply:
Ok